suite-native/link/src/useOpenLink.ts

```diff
@@ -6,19 +6,29 @@ import { useToast } from '@suite-native/toasts';
 export const useOpenLink = () => {
     const { showToast } = useToast();
 
+    const showErrorToast = useCallback(() => {
+        showToast({
+            variant: 'error',
+            icon: 'warningTriangle',
+            message: 'Unable to open the link',
+        });
+    }, [showToast]);
+
     const handleOpenLink = useCallback(
         async (href: string) => {
+            const canOpenURL = await Linking.canOpenURL(href);
```

Comment:
I would rather move it inside the `try` block too, even checking if canOpenURL can fail.

Reply:
1f27a23501